ocs_ci/ocs/resources/packagemanifest.py

```diff
@@ -37,11 +37,23 @@ def __init__(
         )
 
     def get(self, **kwargs):
+        resource_name = kwargs.get("resource_name", "")
```

Comment:
Why not just use `self.resource_name` as the default value here and remove line 41?

```python
resource_name = kwargs.get("resource_name", self.resource_name)
```

Reply:
yep we could :) if I will touch this file I can update this logic here